src/d_clisrv.c

```diff
@@ -3292,7 +3292,7 @@ void SV_StopServer(void)
 	localtextcmd[0] = 0;
 	localtextcmd2[0] = 0;
 
-	for (i = 0; i < BACKUPTICS; i++)
+	for (i = firstticstosend; i < firstticstosend + BACKUPTICS; i++)
 		D_Clearticcmd(i);
```

Comment:
Should note, what you're doing here could equally be achieved by having instead the line below be "D_Clearticcmd(firstticstosend+i);"